shotover-proxy/src/transforms/protect/mod.rs

```diff
@@ -83,36 +242,36 @@ impl From<Protected> for MessageValue {
                 "tried to move unencrypted value to plaintext without explicitly calling decrypt"
             ),
             Protected::Ciphertext { .. } => {
-                MessageValue::Bytes(Bytes::from(serde_json::to_vec(&p).unwrap()))
-                //MessageValue::Bytes(Bytes::from(bincode::serialize(&p).unwrap()))
+                //MessageValue::Bytes(Bytes::from(serde_json::to_vec(&p).unwrap()))
```

Comment:
No point leaving this as a comment, we are in the future now.
We should delete the above TODO as well.